### ISSUE TITLE
CLI confusion with ocdev component create

### ISSUE THREAD

**Issue by cdrage** [redhat-developer/ocdev#255]
The output is this:
```
ocdev component create --help   
create new component

Usage:
  ocdev create [flags]

Flags:
      --binary string   binary artifact
      --dir string      local directory as source
      --git string      git source
  -h, --help            help for create

Global Flags:
  -v, --verbose   verbose output
```

Shouldn't we specify that we pass in a name?

Example:
```
ocdev component create [name] --help   
create new component

Usage:
  ocdev create [flags]

Flags:
      --binary string   binary artifact
      --dir string      local directory as source
      --git string      git source
  -h, --help            help for create

Global Flags:
  -v, --verbose   verbose output
```

**Comment by kadel** [comment]
fix for this is already in PR #247 
 https://github.com/redhat-developer/ocdev/pull/247/files#diff-1dcc122e377b3a7b4adf52faf26c0188R31

**Comment by cdrage** [comment]
@kadel Thanks, didn't see that!
